CODEOWNERS

```diff
@@ -0,0 +1,169 @@
+/src/ @imeckler
+/CODEOWNERS @imeckler @bkase
+/frontend/ @schmavery @bkase
+/dockerfiles/ @jkrauska
+/rfcs/ @imeckler @es92 @bkase
+/docs/ @rbkhmrcr @imeckler @bkase @nholland94 @cmr
+/scripts/ @nholland94 @jkrauska
+/Makefile @jkrauska
+/config/ @imeckler @nholland94
+/CODE_OF_CONDUCT.md @imeckler @cmr
+/CONTRIBUTING.md @imeckler @cmr
+/DIRECTORY_STRUCTURE.md @ihn @jkrauska
+/LICENSE @imeckler
+/README* @cmr
+/azure-pipelines.yml @jkrauska
+
+/src/app/cli/ @cmr @nholland94 @bkase
+/src/app/kademlia-haskell/ @bkase @enolan
+/src/app/lite/ @cmr @bkase
+/src/app/logproc/ @nholland94
+/src/app/reformat/ @bkase
+/src/app/testone/ @johnwu93
+/src/app/trace-tool/ @cmr
+/src/app/website/ @bkase @schmavery
+
+/src/lib/banlist_lib/ @enolan
+/src/lib/bignum_bigint/ @nholland94 @bkase
+/src/lib/blake2/ @rbkhmrcr @vanishreerao
+/src/lib/blockchain_snark/ @nholland94 @bkase
+/src/lib/bootstrap_controller/ @nholland94
+/src/lib/cache_dir/ @cmr
+/src/lib/cached/ @nholland94
+/src/lib/chunked_triples/ @psteckler @rbkhmrcr @vanishreerao
+/src/lib/cli_lib/ @johnwu93
+/src/lib/coda_base/ @cmr @nholland94 @bkase
+/src/lib/coda_base/pending_coinbase* @deepthiskumar
+/src/lib/coda_base/coinbase* @deepthiskumar
+/src/lib/coda_compile_config/ @nholland94
+/src/lib/coda_debug/ @nholland94
+/src/lib/coda_networking/ @cmr
+/src/lib/coda_numbers/ @bkase @nholland94
+/src/lib/coda_state/ @nholland94
+/src/lib/coda_transition/ @nholland94
+/src/lib/codable/ @johnwu93 @bkase
+/src/lib/consensus/ @nholland94
+/src/lib/crs/ @rbkhmrcr @vanishreerao
+/src/lib/crypto_params/ @rbkhmrcr @vanishreerao
+/src/lib/currency/ @bkase @nholland94
+/src/lib/daemon_rpcs/ @johnwu93 @bkase
+/src/lib/debug_assert/ @nholland94 @bkase @cmr
+/src/lib/direction/ @cmr
+/src/lib/distributed_dsl/ @bkase
+/src/lib/dummy_values/ @bkase
+/src/lib/dyn_array/ @johnwu93
+/src/lib/empty_hashes/ @johnwu93
+/src/lib/envelope/ @psteckler @cmr
+/src/lib/external_transition_validation/ @nholland94
+/src/lib/file_stack_database/ @johnwu93 @bkase
+/src/lib/file_system/ @johnwu93 @bkase
+/src/lib/genesis_ledger/ @nholland94 @bkase @cmr
+/src/lib/global_signer_private_key/ @nholland94 # dead code?
+/src/lib/gossip_net/ @cmr
+/src/lib/group_map/ @rbkhmrcr @vanishreerao
+/src/lib/hash_prefixes/ @rbkhmrcr @vanishreerao
+/src/lib/immutable_array/ @nholland94
+/src/lib/interruptible/ @bkase
+/src/lib/kademlia @bkase @cmr @enolan
+/src/lib/key_value_database/ @johnwu93 @bkase
+/src/lib/keys_lib/ @cmr
+/src/lib/ledger_builder/ @deepthiskumar
+/src/lib/ledger_builder_controller/ @nholland94
+/src/lib/ledger_catchup/ @cmr
+/src/lib/ledger_proof/ @nholland94
+/src/lib/linked_tree/ @cmr
+/src/lib/lite_base/ @bkase @cmr
+/src/lib/lite_lib/ @bkase @cmr
+/src/lib/lite_params/ @rbkhmrcr @vanishreerao
+/src/lib/logger/ @nholland94
+/src/lib/merkle_address/ @cmr @bkase
```

Comment:
Like to be a code owner of `merkle_address`. I developed a good amount of the features for this thing and created tests.